packages/system/nfs/Makefile

```
@@ -0,0 +1,3 @@
+export NAME=nfs-system
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)